ui/apps/platform/cypress/integration/vulnerabilities/VulnerabilityReporting/vulnerabilityReportingNav.test.ts

```diff
@@ -0,0 +1,118 @@
+import withAuth from '../../../helpers/basicAuth';
+import {
+    visitVulnerabilityReports,
+    vulnerabilityReportsBasePath,
+} from './VulnerabilityReporting.helpers';
+
+const staticResponseMap = {
+    'reports/configurations': {
+        fixture: 'vulnerabilities/vulnerabilityReporting/vulnerabilityReports.json',
+    },
+};
+
+function interceptVulnerabilityReport(reportId, response, interaction) {
+    cy.intercept('GET', `/v2/reports/configurations/${reportId}`, response).as('reportDetails');
+
+    interaction();
+    cy.wait('@reportDetails');
+}
+
+describe('Vulnerability Reporting Navigation', () => {
+    withAuth();
+
+    it('navigates to the index page', () => {
+        visitVulnerabilityReports(staticResponseMap);
+        cy.location('pathname').should('eq', vulnerabilityReportsBasePath);
+        cy.get('h1').should('contain', 'Vulnerability reporting');
+    });
+
+    it('navigates to the create page', () => {
+        visitVulnerabilityReports(staticResponseMap);
+        cy.contains('button', 'Create report').click();
```

Comment:
What are pro and con here and elsewhere of pseudo-selector `cy.get('button:contains("Create report")')`

A known pro of `contains` method: can have RegExp instead of string for complete match when needed:
* match **central**
* but not match **central**-db